src/jwe/index.ts

```diff
@@ -0,0 +1,45 @@
+import type { JWEHeaderParameters, JWK } from '../types';
+import { IoReactNativeJwt } from '../utils/proxy';
+
+/**
+ * The EncryptJwe class is used to build and encrypt Flattened JWE objects.
+ *
+ * @example Usage
+ *
+ * ```js
+ * const jwe = await new EncryptJwe(
+ *   new TextEncoder().encode('It’s a dangerous business, Frodo, going out your door.', { alg: 'RSA-OAEP-256', enc: 'A256GCM' }),
+ * )
+ *   .encrypt(publicKey)
+ *
+ * console.log(jwe)
+ * ```
+ */
+export class EncryptJwe {
+  private _plaintext: String;
+  private _protectedHeader!: JWEHeaderParameters;
```

Comment:
According to constructor defintion

```suggestion
  private _protectedHeader!: JWEHeaderParameters & { alg: string; enc: string };
```